render/src/simple_element.rs

```diff
@@ -21,7 +21,7 @@ fn write_attributes<'a, W: Write>(maybe_attributes: Attributes<'a>, writer: &mut
         Some(mut attributes) => {
             for (key, value) in attributes.drain() {
                 write!(writer, " {}=\"", key)?;
-                escape_html(&value, writer)?;
+                write!(writer, "{}", value)?;
```

Comment:
Here we stop escaping the html of data that goes into attributes. This does mean library users will need to do this themselves. In future I'd suggest some kind of marker that would escape this *unless* it was attached.